server/Arcadia.Assistant.Server.Console/appsettings.json

```diff
@@ -23,11 +23,33 @@
       "Port": 0,
       "UseTls": true
```

Comment:
Put 'Enabled' here, since you have it in configuration

Reply:
Done.